Extensions/PlatformBehavior/platformerobjectruntimebehavior.ts

```diff
@@ -336,8 +337,12 @@ namespace gdjs {
         this._state.updateFromNetworkSyncData(behaviorSpecificProps.ssd);
       }
 
-      // When the object is synchronized from the network, the inputs must not be cleared.
-      this._dontClearInputsBetweenFrames = true;
+      // When the object is synchronized from the network, the inputs must not be cleared UNLESS we have to force clear them for loading a savestate.
+      if (!options || !options.forceInputClear) {
```

Comment:
you're doing a different logic here than in `PhysicsCharacter3DRuntimeBehavior.ts` 
It ends up being the same, but for consistency, let's do exactly the same if so we can easily search & replace in the codebase :)